types/index.d.ts

```diff
@@ -1,4 +1,4 @@
 export * from './shared'
+export * from './default-theme'
 export * from '../dist/node/index'
 export * from '../dist/client/index'
-export * from '../dist/client/theme-default/config'
```

Comment:
Remove this path since it does not exists: https://unpkg.com/browse/vitepress@0.20.4/dist/client/